src/Tasks/Microsoft.Common.props

```diff
@@ -175,6 +175,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   </PropertyGroup>
   <Import Condition="Exists('$(NuGetPropsFile)')" Project="$(NuGetPropsFile)" />
 
+  <PropertyGroup Condition="'$(GenerateFullPaths)' == ''">
+    <GenerateFullPaths>true</GenerateFullPaths>
+  </PropertyGroup>
```

Comment:
Please move this to Common.CurrentVersion.targets: that's where we set the defaults for most things like this, and it makes the "user set an explicit value in the project file" concept slightly easier.